src/io/aviso/taxi_toolkit/assertions.clj

```diff
@@ -36,9 +36,8 @@
 
 (defn has-class?
   "Indicates whether element has a given class applied"
-  [css-class]
-  (fn [el]
-    (some #{css-class} (classes el))))
+  [css-class el]
```

Comment:
The reason it's returning a function is so that, in `assert-ui` we could do something like:

`(assert-ui {:some-el (has-class? "abc")})`

note that we're invoking the `has-class?` here, and it's returning a anynomous function which is later called internally by `assert-ui` to evaluate validness of an element. That's required because `has-class?` accepts an argument - the class name.

For example:

```clojure
enabled?                   ; we pass function as a parameter, so assert-ui gets a function 
(has-class "class-name")   ; we need to pass a function as well, so has-class? needs to return it
```

I would revert that if there's no other reason for this change.